src/kerberos/client/extractors.rs

```diff
@@ -171,6 +171,7 @@ pub fn extract_status_code_from_krb_priv_response(
 }
 
 /// Extracts [ApRep] from the [NegTokenTarg1] .
+#[instrument(ret)]
```

Comment:
issue: I don’t remember the syntax exactly, but can you verify this is not logging at INFO-level? Use either DEBUG or TRACE in the library.